### ISSUE TITLE
Agenda for January meetup

### ISSUE THREAD

**Issue by nerdyvaishali** [Diversifyingopensource/diversifyingopensource.github.io#2]
I'm creating this to decide the agenda for a next meetup.

- [x] Choose the date -->January 20
- [ ] Confirm the venue
- [ ] Confirm the speaker list

**Comment by amishas157** [comment]
Hi @nerdyvaishali , per chat on phone, I am adding my talk's agenda here:

This talk focuses on two of the most important and widely adopted open data projects, OpenStreetMap and Wikidata, and how we built various tools using semantic query languages and fuzzy matching algorithms to create a symbiotic relationship between these two datasets - to enrich the geo-spatial data on OpenStreetMap with the temporal data from Wikidata. I’ll start off with a brief introduction to both of these projects and subsequently discuss our objectives in bringing these projects together,  the methodologies used to build the tool and how we improved our workflows as a result. I’ll further provide a few examples to show how our workflows have improved as a result of this tool, and in the end conclude with a brief delineation about the significance of a project like this, and open data as a whole.

Also, the above should take around 20 - 25 min.
